demos/storybook/storybook/stories/channel-value.tsx

```diff
@@ -86,7 +87,7 @@ storiesOf('ChannelValue', module)
                     units={text('units', 'hz')}
                     color="error"
                     theme={{
-                        colors: { error: 'pink' },
+                        colors: { error: red[500] },
```

Comment:
red[500] is the default error color in the theme, so setting it to red[500] manually doesn't actually show 'overriding the theme'. You can pick any other color from our palette that passes accessibility (purple, orange?)